src/components/SearchModal/styleds.tsx

```diff
@@ -30,42 +32,47 @@ export const MenuItem = styled(RowBetween)`
   cursor: ${({ disabled }) => !disabled && 'pointer'};
   pointer-events: ${({ disabled }) => disabled && 'none'};
   :hover {
-    background-color: ${({ theme, disabled }) => !disabled && theme.deprecated_bg2};
+    background-color: ${({ theme, disabled, phase0Flag }) =>
+      (phase0Flag && opacify(8, ColorsPalette.gray200)) || (!disabled && theme.deprecated_bg2)};
```

Comment:
all colors should be covered using theme -- should this be theme.backgroundModule? which is opacify(8, ColorsPalette.gray300) -- otherwise should be ok?

Reply:
nah we want gray200, 8 for both light and dark mode as per phil rec. I added it to theme as hoverDefault.